msi/build.sh

```diff
@@ -6,18 +6,35 @@ ENCODEDVERSION="$2"
 ARTIFACTNAME="$3"
 PRODUCTNAME="$4"
 PORT="$5"
-if [ "" == "$PORT" ]; then
-  echo "build.sh path/to/jenkins.war version artifactName port"
+SERVICENAME="$6"
+if [ "" == "$SERVICENAME" ]; then
```

Comment:
Generally this change was not an equivalent if one of parameters is specified. Likely NIT